installer/x86_64/install.sh

```diff
@@ -51,6 +51,10 @@ CONSOLE_SPEED=9600
 
 # Get platform specific linux kernel command line arguments
 ONIE_PLATFORM_EXTRA_CMDLINE_LINUX=""
+
+# Default var/log device size in MB
+VAR_LOG_SIZE=4000
```

Comment:
in case on some platform, they are 128G ssd and have no reason to enable this feature?